app/views/shared/_step_indicator.html.erb

```diff
@@ -0,0 +1,40 @@
+<%#
+locals:
+* class: Extra CSS classes to apply to wrapper.
+* steps: Array of step partial data or step names.
+* locale_scope: Scope under which to find title translations for steps given as names.
+* current_step: Current step.
+%>
+<%
+classes = ['step-indicator']
+classes << local_assigns[:class] if local_assigns[:class]
+normalized_steps = steps.map do |step|
+  step = { name: step } if step.is_a? Symbol
```

Comment:
Not a huge fan of having to type check but it does make it easier than having to type out an array of hashes? 🤷

Reply:
Hm, maybe it'd be simplest for this to only support an array of hashes (vs. supporting either / or / both of symbols and hashes), even if it'd be a bit more verbose.